forum/qa-plugin/block-pw/qa-plugin.php

```diff
@@ -24,26 +25,28 @@
     
 */
 
+declare(strict_types=1);
+
 if (!defined('QA_VERSION')) {
     header('Location: ../../');
     exit;
 }
 
-qa_register_plugin_phrases('block-pw-lang-*.php', 'block_pw');
-qa_register_plugin_module('page', 'block-pw-page.php', 'block_pw_page', 'Block pw page');
-qa_register_plugin_module('page', 'block-pw-user-list-page.php', 'block_pw_user_list_page', 'Block pw user list page');
-qa_register_plugin_overrides('block-pw-override.php');
-qa_register_plugin_layer('block-pw-layer.php', 'Block pw layer');
+qa_register_plugin_phrases('block-pm-lang-*.php', 'block_pm');
+qa_register_plugin_module('page', 'block-pm-page.php', 'block_pm_page', 'Block pm page');
+qa_register_plugin_module('page', 'block-pm-user-list-page.php', 'block_pm_user_list_page', 'Block pm user list page');
+qa_register_plugin_overrides('block-pm-override.php');
+qa_register_plugin_layer('block-pm-layer.php', 'Block pm layer');
+qa_register_plugin_module('module', 'block-pm-admin.php', 'block_pm_admin', 'Block pm admin');
 
 /**
  * Parameters order does not matter.
  */
-function checkIfUserIsBlocked($fromUserId, $toUserId): bool
+function ifUserIsBlocked(int $fromUserId, int $toUserId): bool
```

Comment:
to może sprawić jednak problem - upewniłbym się czy na przykład po wylogowaniu nie rzuca Ci błędami :) - szczególnie jak próbujesz się ponowaine zalogować.